WcaOnRails/app/webpacker/components/CompetitionForm/index.js

```diff
@@ -318,7 +319,9 @@ export default function Wrapper({
       }}
     >
       <SectionProvider>
-        <CompetitionForm />
+        <QueryClientProvider client={new QueryClient()}>
```

Comment:
One nit is to move the `new QueryClient()` into a const outside of Wrapper that way this object isn't being re-instantiated every time there's a re-render of Wrapper.